gapic/src/main/com/google/gapid/widgets/Widgets.java

```diff
@@ -24,6 +24,7 @@
 import com.google.common.util.concurrent.SettableFuture;
 import com.google.gapid.models.Models;
 import com.google.gapid.server.Client;
+import com.google.gapid.views.AtomEditor;
```

Comment:
Yes. The class was moved out of this package into the views package, thus it now needs to be imported.